policyengine_us/parameters/gov/states/ca/cdss/tanf/resources_limit/lower.yaml

```
@@ -0,0 +1,11 @@
+description: California TANF limits households with up to this amount of resources if they have no elderly or disabled member.
```

Let's call it `CalWORKS` in all descriptions and labels and phrase it like this
```suggestion
description: California limits CalWORKS to households with up to this amount of resources if they have no elderly or disabled member.
```